assets/js/src/form_editor/blocks/custom_textarea/edit.jsx

```diff
@@ -159,10 +159,14 @@ const CustomTextAreaEdit = ({
 
   const textarea = useRef(null);
   const getTextArea = (placeholder) => {
-    let style = '';
+    let style = `#${id}::placeholder {`;
```

Comment:
Just an idea. Because of placeholder, we render <style> block with every component (input, textarea) and we use the component id in CSS selector. So I'm thinking that we could use <[FormStyles](https://github.com/mailpoet/mailpoet/blob/master/assets/js/src/form_editor/components/form_styles.jsx)> component (currently used only for Custom CSS rendering) and render these rules inside this component using block class names in a selector. This could be possibly used for all form global styles. What do you think?

Reply:
Good idea.